spec/haskell/src/SEL4/Machine/Hardware/RISCV64.hs

```diff
@@ -336,11 +336,15 @@ maxUsToTicks = undefined
 maxTicksToUs :: Word64
 maxTicksToUs = undefined
 
+maxPeriodUs :: Word64
+maxPeriodUs = undefined
+
 debugPrint :: String -> MachineMonad ()
 debugPrint str = liftIO $ putStrLn str
 
 read_sbadaddr :: MachineMonad Word
 read_sbadaddr = error "Unimplemented - machine op"
 
 plic_complete_claim :: IRQ -> MachineMonad ()
-plic_complete_claim = error "Unimplemented - machine op"
```

Comment:
Oh I thought that GitHub was complaining that I had deleted the line at the end of the file, so I added one. Now I'm not sure what I should have done...

Reply:
Add this new line made the trailing whiespace test fail. So I have gone back to having no new line at the end of the file. So confused

Reply:
Previously this file did not end in a newline character. However, the standard convention is that it should so my best guess is that your editor is automatically adding one whenever you edit and save the file.

I'd say that you can leave it as is, the newline has been added and apart from the spurious change in this commit it shouldn't come up again in the future.

Reply:
Has the newline been added? I opened up the source for this file and it looks like there's no new line at the end. Maybe that view just doesn't display it. The tests are passing as it is, and what I did before did break them, so I guess I can just leave it as unless there are other issues